grpc/src/main/resources/assets/pt/src/view/map/Map.js

```diff
@@ -61,8 +61,11 @@ class LeafletComponent extends React.Component {
       iconSize: 10
     }));
     let bbox = this.props.info.bbox;
-    console.log(bbox)
-    this.map.fitBounds(Leaflet.latLngBounds(Leaflet.latLng(bbox[1], bbox[0]), Leaflet.latLng(bbox[3], bbox[2])));
+    if (this.props.from != null && this.props.to != null) {
+        bbox = [this.props.from.long, this.props.from.lat, this.props.to.long, this.props.to.lat];
+    }
+    console.log(bbox);
+    this.map.fitBounds(Leaflet.latLngBounds(Leaflet.latLng(bbox[1], bbox[0]), Leaflet.latLng(bbox[3], bbox[2])), {padding: [50, 50]});
```

Comment:
is it fine to add this padding for the default case where there's no from/to? probably not noticeable, but just wanted to check

Reply:
Good q! From my testing this didn't really seem to be an issue, the default zoom is just slightly further out than it would normally be